app/posts/manage-teacher-training-applications/2023-06-01-provider-recruitment-performance.md

```diff
@@ -0,0 +1,59 @@
+---
+
+![Illustration with the heading '2022 to 2023 recruitment cycle performance' followed by a table with 3 columns: Provider A, split by this cycle, last cycle and pecentage change. Then all providers, split by this cycle, last cycle and percentage change.](mid-cycle-cover.png)
+
+We released 2 reports to show training providers how their recruitment performance was going halfway through the recruitment cycle (from 11 October 2022 to 3 April 2023).
+
+ One report showed training providers their recruitment performance compared to the national recruitment figures.
+
+ The other report showed the overall national recruitment performance so far this recruitment cycle.
+
+ These reports were released on 22 May 2023.
+
+## The issue
+
+So far this recruitment cycle, we’ve seen a higher rate of rejections being issued by training providers. On one day we saw 718 applications rejected automatically.
+
+ We worked with our policy team to see what we could do to improve this. Our policy team were already working on sending a letter to training providers from the Permanent Secretary. These reports would accompany the letter so that providers could positively engage in their own data and see for themselves where recruitment improvements could be made.
+
+## What we changed
+
+We added a new report showing training providers their own recruitment data from 11 October 2022 to 3 April 2023. The report includes 8 tables, which are:
+
+- candidates who’ve submitted applications
+- overall number of applications
+- applications with an offer
+- offers accepted
+- deferrals
+- applications rejected
+- applications rejected automatically
+- proportion of applications rejected
+
+ The tables also show data from national recruitment figures so providers can easily see how they are performing against the national recruitment picture. This report is available from within the Manage teacher training applications service.
+
+ Where a provider offers Secondary subjects, we show a breakdown of the data per subject.
+
+ The data in the image is fake, and used for testing purposes.
+
+![Screenshot showing a page titled '2022 to 2023 recruitment cycle performance'.](provider-mid-cycle-report.png)
+
+We added another [report that shows the national figures](https://www.apply-for-teacher-training.service.gov.uk/publications/mid-cycle-report) that we published independently from Manage. This is because there may be other people interested in national recruitment data.
+
+![Screenshot showing a page titled 'National recruitment performance for the 2022 to 2023 recruitment cycle'.](national-mid-cycle-report.png)
+
+## What we learnt
+
+On the first day we realsed this report, we received 108 unique page views from providers. Between the 22 May 2023 and 24 May 2023 this number increased to 322 providers. This shows providers are engaging with their own data.
```

Comment:
Are these 2 numbers counting different things (unique page views vs unique providers)? It’s not quite clear.

Reply:
Ah yeah. I think I read the data wrong. It's providers rather than page views.